README.md

```diff
@@ -1,8 +1,10 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+contact form project
 
 > Description the project.
+> This is simple portfolio project
 
 
```

Comment:
- [ ] Please customize the README template to your project information. To be specific, please add the project name and description. This makes your portfolio professional